include/aspect/velocity_boundary_conditions/interface.h

```diff
@@ -59,12 +59,23 @@ namespace aspect
          * Initialization function. This function is called once at the
          * beginning of the program after parse_parameters is run and after
          * the SimulatorAccess (if applicable) is initialized.
+         *
+         * @depricated Use initialize(types::boundary_id boundary_id) instead.
          */
         virtual
         void
         initialize ();
```

Comment:
Ah sorry, you already went through the plugins. Did you check the benchmarks, cookbooks and tests as well? The pain of making backward incompatible changes :smirk: